### PR TITLE
update travis.yml

### PR DESCRIPTION
**Pull request by galgeek**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,33 @@
...
+  - TOX_ENV=py36-cdh
+  - TOX_ENV=py36-hdp
+- 3.8
+  env:
+  - TOX_ENV=py38-cdh
+  - TOX_ENV=py38-hdp
+jobs:
+  allow_failures:
+    python:
+    - "3.4"
```
Comment (cclauss) [inline .travis.yml:22]: Why continue to run Py34?
* https://devguide.python.org/#status-of-python-branches
* https://devguide.python.org/devcycle/#end-of-life-branches

File: .travis.yml [hunk 2]
```diff
@@ -1,15 +1,33 @@
...
+    - "3.8"
 jdk: oraclejdk7
 sudo: false
```
Comment (cclauss) [inline .travis.yml:25]: The `sudo` tag is deprecated in Travis CI.
```suggestion
```

File: .travis.yml (hunk 1 above)
Comment (galgeek) [reply .travis.yml:22]: I added back 3.4 after noticing 3.4 was still present in tox.ini and wondering if others had a use for it.

**Comment by galgeek** [comment]
closing, travis is obsolete here

**Pull request closed without merging**